provider/maas/environ_whitebox_test.go

```diff
@@ -1759,19 +1762,9 @@ func (s *environSuite) TestStartInstanceDistribution(c *gc.C) {
 	s.testMAASObject.TestServer.AddZone("test-available", "description")
 	s.newNode(c, "node1", "host1", map[string]interface{}{"zone": "test-available"})
 	inst, _ := testing.AssertStartInstance(c, env, "1")
-	c.Assert(inst.(*maasInstance).zone(), gc.Equals, "test-available")
-}
-
-func (s *environSuite) TestStartInstanceDistributionAZNotImplemented(c *gc.C) {
```

Comment:
Yes, this is my current outstanding question. I will be clarifying this before landing the PR. 

Of course, if we still support MAAS versions without AZ, this PR will change and this test re-instated.